packages/app-accounts/src/ChangePass.tsx

```diff
@@ -127,16 +133,35 @@ class ChangePass extends React.PureComponent<Props, State> {
       account.decodePkcs8(oldPass);
     } catch (error) {
       this.setState({ isOldValid: false });
+
+      status.message = t('status.error', {
+        defaultValue: error.message
+      });
+
       return;
     }
 
     try {
       keyring.encryptAccount(account, newPass);
+
+      status.value = account.address();
+      status.success = true;
+      status.message = t('status.change-password', {
+        defaultValue: 'Password Changed'
+      });
     } catch (error) {
       this.setState({ isNewValid: false });
+
+      status.success = false;
+      status.message = t('status.error', {
+        defaultValue: error.message
```

Comment:
Here is doesn't make sense to have it translated, i.e. can just pass the error.message through